customizations/README.md

```diff
@@ -0,0 +1,187 @@
+# Customizations
+
+That file explain how to configure your CARTO selfhosted to meet your needs.
+
+## Production Ready
+
+By default, the Helm configuration with the config files provided by CARTO should work but it's not production ready.
+In the following steps we are going to explain the steps to approximate the installation to a production ready environment.
+
+It should be configured:
+- **MANDATORY** Configure the domain to be used.
+- Allow the external traffic to access the app.
+- Configure DBs to be production ready. Our recomendation is to use managed DBs with backups and so on.
+
+Configuring it is optional:
+- Configure scale of the components (staticaly or with auto-scaling).
+- Use your own bucket to store the data (By default, GCP CARTO buckets are used)
+
+## Architecture diagram
+
+<!--
+We should add an arquitectural diagram to make it easier for customers to understand the parts and the relationship between them.
+-->
+
+## How to define customizations
+
+There are two ways to configure or customize the deployment:
+- [**RECOMMENDED**] Create a dedicated [yaml](https://yaml.org/) file. For example, you can create a file with the next content:
+  ```yaml
+  customConfigValues:
+    selfHostedDomain: "my.domain.com"
+  ```
+  And add the following at the end of ALL the install or upgrade command:
+  ```bash
+  ... -f <my_customization_file>.yaml
+  ```
+- Use the inline set. For example, add the following at the end of ALL the install or upgrade command:
+  ```bash
+  ... --set customConfigValues.selfHostedDomain=my.domain.com
+  ```
+
+
+## Configure the domain of your selfhosted
+
+The most important step to have your CARTO self-hosted ready to be used is to configure the domain to be used.
+
+⚠️ CARTO selfhosted is not designed to be used in the path of a URL, it needs a full domain or subdomain. ⚠️
+
+To do this you need to [add the following customization](#how-to-define-customizations):
+```yaml
+customConfigValues:
+  selfHostedDomain: "my.domain.com"
+```
+
+Don't forget to upgrade your chart after the change.
+
+
+## Access to CARTO from outside the cluster
+By default, the `router` Service of CARTO selfhosted is configured in mode `ClusterIP` so it's only usable from inside the cluster.
+To access to it you can exec locally a [kubectl port-forward](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/) but this only make it accessible to your machine.
+
+To made it accessible from the internet (or outside the Kubernetes network) you need to properly configure the `router` Service (the easiest way) or create an Ingress.
```

Comment:
I would skip the whole Ingress so far. Even say that so far the only supported way is through a `LoadBalancer`

Reply:
Pufff. Currently I only prepare it but I notice some shortcomings such as not being able to define a static IP (or at least I didn't see it for EKS).

At the moment I did not write anything and I am not going to do it as we said in the daily.

You decide if I remove it or leave it commented until we see what to do or leave it like this.